### PR TITLE
Browser fixture

### PR DESCRIPTION
**Pull request by cleborys**
`driver_list` fixtures enables tests to easily work with several browsers by wrapping everything in `for driver in driver_list`. 
This is not ideal, as a failing test in _any_ browser will cause the test for _all_ browsers to fail, see issue #1.

### CODE REVIEW

**Comment by FjolleJagt** [comment]
You're currently merging this into master; could you merge this into create_dummy_user instead, so I can see a diff with that? (Or merge the other pull request into master and leave this as-is.)

File: login_test.py [hunk 1]
```diff
@@ -96,50 +96,58 @@ def dummy_user():
 
 
 @pytest.fixture
-def firefox_driver(request, scope = 'function'):
+def driver_list(request, scope = 'function'):
```
Comment (FjolleJagt) [inline login_test.py]: I don't think we actually need to have this as a fixture, do we? It's not like it does any setup behind the scenes, it is just a standard python generator function.

**Review by FjolleJagt — APPROVED** [review]
This seems a good fix for now. Happy this being merged once the above comment is addressed.

File: login_test.py (hunk 1 above)
Comment (cleborys) [reply login_test.py]: The point of having a fixture returning a generator was that the fixture is run to the end after the test shut down and I wanted to be sure that `driver.quit()` is also run for the last driver in any test.
However, a) looking at it I'm not sure that this is even the case in the current implementation and b) it seems that selenium webdrivers have `__enter__` and `__exit__` methods, so `with` statements are probably the way to go. Will change that.

**Comment by cleborys** [comment]
Creating the dummy user seems broken when running from docker, I am not sure how to connect to the database. Thus all "positive" tests assuming that there is a valid user are failing when running in docker.

**Comment by FjolleJagt** [comment]
Seems fine to me; do you still have the problems with creating the dummy user, or is that now fixed by exposing the db? I would prefer if we only had passing tests on master.

**Review by FjolleJagt — APPROVED** [review]

**Comment by cleborys** [comment]
> Seems fine to me; do you still have the problems with creating the dummy user, or is that now fixed by exposing the db? I would prefer if we only had passing tests on master.

That was the problem, yes. This now works, as long as parentnode_docker with the version that exposes the database on the appropriate port is used.

**Comment by cleborys** [comment]
(Just verified that everything is passing in my setup)

**Pull request merged**